src/main/java/io/aiven/kafka/connect/http/sender/HttpSender.java

```diff
@@ -20,4 +20,6 @@
 
 public interface HttpSender {
     HttpResponse<String> send(final String body);
+
+    HttpResponse<String> send(final String body, String key);
```

Comment:
Consider passing the key as a function instead of a value:
```suggestion
    HttpResponse<String> send(final String body, Supplier<String> keySupplier);
```